### PR TITLE
External DNS template

### PR DESCRIPTION
**Pull request by alena1108**
@cloudnautique Bill, can you please review. 

We should hold off merging as the image wasn't pushed to dockerhub yet.

### CODE REVIEW

File: templates/external-dns/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      type: "string"
+      required: true
+    - variable: "AWS_SECRET_KEY"
+      label: "AWS secret key"
+      description: "Secret key to your AWS account"
+      type: "string"
+      required: true
+    - variable: "AWS_REGION"
+      label: "AWS region"
+      description: "AWS regional endpoint for DNS configuration requests"
```
Comment (cloudnautique) [inline templates/external-dns/0/rancher-compose.yml]: Needs a type?

File: templates/external-dns/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.catalog:
+  name: "Route53 DNS"
+  version: "v0.1.2"
```
Comment (cloudnautique) [inline templates/external-dns/0/rancher-compose.yml]: We have been adding -rancherX for a build number... that way we can change the templates without incrementing the version.

Comment (alena1108) [inline templates/external-dns/0/rancher-compose.yml]: @cloudnautique will fix to be "v0.44.0"

**Comment by cloudnautique** [comment]
LGTM - merging

**Pull request merged**